scripts/setup_env

```diff
@@ -32,16 +32,16 @@ USE_VENV=${USE_VENV:-'yes'}
 source /etc/os-release || source /usr/lib/os-release
 RHT_PKG_MGR=$(command -v dnf || command -v yum)
 PYTHON_EXEC=$(command -v python3 || command -v python)
```

Comment:
i generally appreciate this form, but in this case isn't a guarantee to use system python (e.g. different python version installed in path)

i'd switch to `/usr/bin/python3` avoiding `python` (which should be deprecated in EL9)